pages/roadmap.vue

```diff
@@ -19,8 +19,11 @@
                         <i class="far fa-external-link" />
                       </a>
                     </h5>
-                    <ul class="items">
-                      <li v-for="(item, l) in goal.list" :key="l">{{item}}</li>
+                    <ul>
+                      <li v-for="(item, l) in goal.list" :key="l" class="icon-list" mb1>
+                        <span></span>
```

Comment:
again here, useless span, a span shouldn't be used to display a bullet style in a list

Reply:
yes but it's the only solution i found to have it work, with a before this is not working. I look at other website to find this solution and for examples Prisma or Zapier are actually using this.